gc/base/GCExtensionsBase.hpp

```diff
@@ -1870,6 +1872,8 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		, scavengerAlignHotFields(true) /* VM Design 1774: hot field alignment is on by default */
 		, suballocatorInitialSize(SUBALLOCATOR_INITIAL_SIZE) /* default for J9Heap suballocator initial size is 200 MB */
 		, suballocatorCommitSize(SUBALLOCATOR_COMMIT_SIZE) /* default for J9Heap suballocator commit size is 50 MB */
+		, suballocatorIncrementSize(SUBALLOCATOR_INCREMENT_SIZE) /* default for J9Heap suballocator commit size is 8 MB or 256 MB for AIX */
+		, suballocatorQuickAlloc(true) /* use mmap-based allocation by default for the J9Heap suballoctor */
```

Comment:
```suggestion
		, suballocatorQuickAlloc(true) /* use mmap-based allocation by default for the J9Heap suballocator */
```

Reply:
OMR shouldn't use J9. Can `J9Heap` be changed to `heap` (something more generic)?

Reply:
Sure